requirements.txt

```diff
@@ -10,4 +10,6 @@ boto3>=1.12.0
 psycopg2
 psutil
 discordhealthcheck
-discord-py-interactions
\ No newline at end of file
+discord-py-interactions==3.0.2 # They keep rewriting their codebase and I'm sick of it
+python-Levenshtein
```

Comment:
What's this import do?

Reply:
This speeds up fuzzywuzzy, it's not required but I feel it's worth including.